whisper_online.py

```diff
@@ -611,14 +622,18 @@ def asr_factory(args, logfile=sys.stderr):
     logfile = sys.stderr
 
     if args.offline and args.comp_unaware:
-        print("No or one option from --offline and --comp_unaware are available, not both. Exiting.",file=logfile)
+        logger.error("No or one option from --offline and --comp_unaware are available, not both. Exiting.")
         sys.exit(1)
 
+    if args.log_level:
+        logging.basicConfig(format='whisper-%(levelname)s:%(name)s: %(message)s',
+                            level=getattr(logging, args.log_level))
```

Comment:
I think we should always call `basicConfig`. Because of the default, `log_level` is set anyway.